dashboard/src/modules/components/RelayUIComponent/index.jsx

```diff
@@ -77,10 +77,17 @@ const RelayComponent = () => {
         <PopoverComponent />
       </div>
       <p className="modal-desc">
-        The Pbench Agent can push datasets to a Pbench Server directly, when the
-        server is accessible from the system where the Pbench Agent is being
-        used.
+        When you use the Pbench Agent{" "}
+        <span className="italicized"> pbench-results-move </span> command with
+        the
+        <span className="italicized"> --relay </span>option to push results to a
```

Comment:
Commands and other terminal input are traditionally displayed in fixed-width normal font, not italics.  (I presume that "italicized" here is just an arbitrary label, but the corresponding definition for it in the `.less` file seems to be setting it to `italic`.)  A reference that I found recommended using the `<kbd>` HTML tag or CSS `font-family: monospace`.